BubblesKotlin/app/src/main/java/com/example/android/bubbles/MainActivity.kt

```diff
@@ -71,10 +72,20 @@ class MainActivity : AppCompatActivity(), NavigationController {
     }
 
     private fun handleIntent(intent: Intent) {
-        if (intent.action == Intent.ACTION_VIEW) {
-            val id = intent.data?.lastPathSegment?.toLongOrNull()
-            if (id != null) {
-                openChat(id)
+        when (intent.action) {
```

Comment:
Maybe add a small bit of inline doc here to explain where these intents come from, etc.

Reply:
Done.